galasa-parent/dev.galasa.framework.api.ras/openapi.yaml

```diff
@@ -130,6 +130,11 @@ paths:
       tags:
         - RasRun
       parameters:
+        - name: sort
+          in: query
+          description: provides sorting, timeend:asc or timeend:desc
```

Comment:
example should really use an actual parameter,    to:asc or to:desc    would be technically correct